### PR TITLE
feat: add holesky support, update doc

### PR DESCRIPTION
**Pull request by BATMAH69**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,12 +70,20 @@ git clone git@github.com:lidofinance/AVotesParser.git
...
 cd production/avotes-parser-core
+poetry run pip install "cython<3.0" pyyaml==5.4.1 --no-build-isolation
 poetry install
 
 cd ../avotes-parser-cli
+poetry run pip install "cython<3.0" pyyaml==5.4.1 --no-build-isolation
 poetry install
 ```
 
+### for development
```
Comment (TheDZhon) [inline README.md:81]: ♥

**Review by katamarinaki — APPROVED** [review]
👍

File: README.md [hunk 2]
```diff
@@ -88,18 +96,24 @@ poetry shell
...
+  --aragon-voting-address ARAGON_VOTING_ADDRESS
+                        Address of aragon voting contract. (default: 0x2e59A20f205bB85a89C53f1936454680651E618e)
+  
+optional arguments:
+  -h, --help            show this help message and exit
   -n N                  Parse last N votes. (default: 10)
   --aragon-voting-address ARAGON_VOTING_ADDRESS
                         Address of aragon voting contract. (default: 0x2e59A20f205bB85a89C53f1936454680651E618e)
-  --net {mainnet,goerli,kovan,rinkebay,ropsten}
+  --net {mainnet,goerli,holesky,kovan,rinkebay,ropsten}
```
Comment (TheDZhon) [inline README.md]: let's leave only mainnet, goerli and holesky

**Review by TheDZhon — CHANGES_REQUESTED** [review]
Thank you so much :+1:

I'd suggest to remove the deprecated networks to not blow up their list :)

File: production/avotes-parser-core/avotes_parser/core/ABI/utilities/etherscan.py [hunk 3]
```diff
@@ -21,6 +21,7 @@
 NET_URL_MAP = {
     'mainnet': 'https://api.etherscan.io',
     'goerli': 'https://api-goerli.etherscan.io',
+    'holesky': 'https://api-holesky.etherscan.io',
     'kovan': 'https://api-kovan.etherscan.io',
```
Comment (TheDZhon) [inline production/avotes-parser-core/avotes_parser/core/ABI/utilities/etherscan.py]: can remove the deprecated networks :pray:

**Review by TheDZhon — APPROVED** [review]
Let's roll it out

**Pull request merged**